src/copy/index.ts

```diff
@@ -15,6 +15,7 @@ export const copy = {
         bottom: "FREE - On the App Store",
       },
     },
+    footer: { companyName: "The Place.", msg: "All rights reserved." },
```

Comment:
Under footer add a property called socials and initialise it with an array of socials (strings).